### PR TITLE
Portfolio: setup and mobile first

### PR DESCRIPTION
**Pull request by TumiRantsane**
I've created HTML templates and styling files in order to build my responsive portfolio mobile version and desktop version.

### CODE REVIEW

**Review by victorokoroji — COMMENTED** [review]
Hi @TumiRantsane, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## ❌ Invalid Code Review Request

You have submitted a project, that does not have a well-written and descriptive readme. Kindly add fill up your readme sections and submit again 👍🏽 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by TumiRantsane** [comment]
Hello Victor. I have setup the Readme section, can you kindly review. Sent from Mail for Windows From: Victor Ebube OkorojiSent: Saturday, 29 October 2022 9:10 PMTo: TumiRantsane/Portfolio-SetupCc: TumiRantsane; MentionSubject: Re: [TumiRantsane/Portfolio-Setup] Portfolio: setup and mobile first (PR ***@***.*** commented on this pull request.Hi @TumiRantsane,While you made a great effort in this project, unfortunately, I cannot proceed to review your code.❌ Invalid Code Review RequestYou have submitted a project, that does not have a well-written and descriptive readme. Kindly add fill up your readme sections and submit again 👍🏽Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏Cheers and Happy coding!👏👏👏Feel free to leave any questions or comments in the PR thread if something is not 100% clear.Please, remember to tag me in your question so I can receive the notification.Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.Invalid Code Review Request does not count into the code reviews limit.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Review by Lizdev-05 — COMMENTED** [review]
Hi @TumiRantsane, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a READme file, kindly use this [READme](https://github.com/microverseinc/readme-template/blob/master/README.md) to create your file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                <div>
+                   <ul>
+                     <li class="icon"><a href="https://twitter.com/irantsane"><img src="images/twitter.png" alt="twitter link"></a></li>
+                     <li class="icon"><a href="https://www.linkedin.com/in/itumeleng-rantsane-552a44244/"><img src="images/Linkedin_icon.png" alt="linked link"></a></li>
+                     <li class="icon"><a href=""><img src="images/M.png" alt="M"></a> </li>
+                     <li class="icon"><a href="https://github.com/TumiRantsane"><img src="images/github.png" alt="github link"></a></li>
+                     <li class="icon"><a href=""><img src="images/doits.png" alt="doit"></a></li>
+                   </ul>
+                </div>
+            </footer>
```
Comment (Abenezer-Tilahun) [inline index.html:40]: - [ ] Kindly remove the `footer` tag , your social icons list are part of "Headline" section So Please try to remove the `footer` tag.

File: styles/style.css [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+section h1 {
+  font-family: Poppins, sans-serif;
+  font-weight: bold;
+  font-size: 40px;
+  color: #172b4d;
+  line-height: 52px;
+  text-align: start;
+  margin: 0 24px;
+}
```
Comment (Abenezer-Tilahun) [inline styles/style.css:72]: - [ ] You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Kindly Adjust the background image alignment just like design and adjust font-weight of main title("I'm Itumeleng. So glad to see you!") and "Let's connect " Just Like figma design.

Your Project Look likes this : 
Capture

The Figma design Look likes this : 
Capture

**Review by Abenezer-Tilahun — COMMENTED** [review]
Hi @TumiRantsane  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Professional  README 
✔️ Good commit messages

almost

## Required Changes ♻️

- [ ] Kindly I recommend you to add "Gets started" header section at your README file.

Capture

_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**